cape-ph/cape-cod
# add baseline cape analysis UI application to repo

# TO TEST
* `pulumi preview` should show no differences to what's currently deployed
* This is currently deployed and test can be run by:
  * getting on cape vpn
  * hitting `https://analysis-pipelines.cape-dev.org` to see the new baseline site
  * hitting `https://analysis-pipelines.cape-dev.org/nested` to see the previous toy root page
  * hittng `https://analysis-pipelines.cape-dev.org/nested/request-dap` to see the previous request dap page
